From bcef1eb45d251d41721417378a865e8a89a3c12a Mon Sep 17 00:00:00 2001
From: Fabian Greffrath <fabian@greffrath.com>
Date: Mon, 2 Oct 2017 11:08:26 +0200
Subject: [PATCH 1/3] appstream: fix typos and syntax errors

 * "update_conctact" should be "update_contact"
 * XML comments are closed with "-->", i.e. without an extra
   exclamation mark

Fixes https://github.com/ArtifexSoftware/urw-base35-fonts/issues/10
---
 appstream/de.urwpp.C059.metainfo.xml                 |  4 ++--
 appstream/de.urwpp.D050000L.metainfo.xml             |  4 ++--
 appstream/de.urwpp.NimbusMonoPS.metainfo.xml         |  4 ++--
 appstream/de.urwpp.NimbusRoman.metainfo.xml          |  4 ++--
 appstream/de.urwpp.NimbusSans.metainfo.xml           |  4 ++--
 appstream/de.urwpp.NimbusSansNarrow.metainfo.xml     |  4 ++--
 appstream/de.urwpp.P052.metainfo.xml                 |  4 ++--
 appstream/de.urwpp.StandardSymbolsPS.metainfo.xml    |  4 ++--
 appstream/de.urwpp.URWBookman.metainfo.xml           |  4 ++--
 appstream/de.urwpp.URWCoreFontSetLevel2.metainfo.xml | 10 +++++-----
 appstream/de.urwpp.URWGothic.metainfo.xml            |  4 ++--
 appstream/de.urwpp.Z003.metainfo.xml                 |  4 ++--
 12 files changed, 27 insertions(+), 27 deletions(-)

diff --git a/appstream/de.urwpp.C059.metainfo.xml b/appstream/de.urwpp.C059.metainfo.xml
index a022a0e..318e115 100644
--- a/appstream/de.urwpp.C059.metainfo.xml
+++ b/appstream/de.urwpp.C059.metainfo.xml
@@ -9,7 +9,7 @@
   <url type="homepage">https://www.urwpp.de/en/</url>
   <url type="bugtracker">https://bugs.ghostscript.com/</url>
 
-  <update_conctact>dkaspar@redhat.com</update_contact>
+  <update_contact>dkaspar@redhat.com</update_contact>
 
   <name>C059</name>
   <summary>An alternative font family for New Century Schoolbook typeface</summary>
@@ -45,7 +45,7 @@
     <id>de.urwpp.Z003</id>
   </suggests>
 
-  <!-- NOTE: Medium urgency of updates is implicitly assumed with AppStream --!>
+  <!-- NOTE: Medium urgency of updates is implicitly assumed with AppStream -->
   <releases>
     <release version="20170801" date="2017-08-01" />
     <release version="20160926" date="2016-09-26" />
diff --git a/appstream/de.urwpp.D050000L.metainfo.xml b/appstream/de.urwpp.D050000L.metainfo.xml
index 107519b..dce5db7 100644
--- a/appstream/de.urwpp.D050000L.metainfo.xml
+++ b/appstream/de.urwpp.D050000L.metainfo.xml
@@ -9,7 +9,7 @@
   <url type="homepage">https://www.urwpp.de/en/</url>
   <url type="bugtracker">https://bugs.ghostscript.com/</url>
 
-  <update_conctact>dkaspar@redhat.com</update_contact>
+  <update_contact>dkaspar@redhat.com</update_contact>
 
   <name>D050000L</name>
   <summary>An alternative font for ITC Zapf Dingbats typeface</summary>
@@ -42,7 +42,7 @@
     <id>de.urwpp.Z003</id>
   </suggests>
 
-  <!-- NOTE: Medium urgency of updates is implicitly assumed with AppStream --!>
+  <!-- NOTE: Medium urgency of updates is implicitly assumed with AppStream -->
   <releases>
     <release version="20170801" date="2017-08-01" />
     <release version="20160926" date="2016-09-26" />
diff --git a/appstream/de.urwpp.NimbusMonoPS.metainfo.xml b/appstream/de.urwpp.NimbusMonoPS.metainfo.xml
index 5db4919..4cdf5e1 100644
--- a/appstream/de.urwpp.NimbusMonoPS.metainfo.xml
+++ b/appstream/de.urwpp.NimbusMonoPS.metainfo.xml
@@ -9,7 +9,7 @@
   <url type="homepage">https://www.urwpp.de/en/</url>
   <url type="bugtracker">https://bugs.ghostscript.com/</url>
 
-  <update_conctact>dkaspar@redhat.com</update_contact>
+  <update_contact>dkaspar@redhat.com</update_contact>
 
   <name>Nimbus Mono PS</name>
   <summary>An alternative font family for Courier typeface</summary>
@@ -45,7 +45,7 @@
     <id>de.urwpp.Z003</id>
   </suggests>
 
-  <!-- NOTE: Medium urgency of updates is implicitly assumed with AppStream --!>
+  <!-- NOTE: Medium urgency of updates is implicitly assumed with AppStream -->
   <releases>
     <release version="20170801" date="2017-08-01" />
     <release version="20160926" date="2016-09-26" />
diff --git a/appstream/de.urwpp.NimbusRoman.metainfo.xml b/appstream/de.urwpp.NimbusRoman.metainfo.xml
index 447857f..a085577 100644
--- a/appstream/de.urwpp.NimbusRoman.metainfo.xml
+++ b/appstream/de.urwpp.NimbusRoman.metainfo.xml
@@ -9,7 +9,7 @@
   <url type="homepage">https://www.urwpp.de/en/</url>
   <url type="bugtracker">https://bugs.ghostscript.com/</url>
 
-  <update_conctact>dkaspar@redhat.com</update_contact>
+  <update_contact>dkaspar@redhat.com</update_contact>
 
   <name>Nimbus Roman</name>
   <summary>An alternative font family for Times New Roman typeface</summary>
@@ -45,7 +45,7 @@
     <id>de.urwpp.Z003</id>
   </suggests>
 
-  <!-- NOTE: Medium urgency of updates is implicitly assumed with AppStream --!>
+  <!-- NOTE: Medium urgency of updates is implicitly assumed with AppStream -->
   <releases>
     <release version="20170801" date="2017-08-01" />
     <release version="20160926" date="2016-09-26" />
diff --git a/appstream/de.urwpp.NimbusSans.metainfo.xml b/appstream/de.urwpp.NimbusSans.metainfo.xml
index 1aa3468..28d9c96 100644
--- a/appstream/de.urwpp.NimbusSans.metainfo.xml
+++ b/appstream/de.urwpp.NimbusSans.metainfo.xml
@@ -9,7 +9,7 @@
   <url type="homepage">https://www.urwpp.de/en/</url>
   <url type="bugtracker">https://bugs.ghostscript.com/</url>
 
-  <update_conctact>dkaspar@redhat.com</update_contact>
+  <update_contact>dkaspar@redhat.com</update_contact>
 
   <name>Nimbus Sans</name>
   <summary>An alternative font family for Helvetica typeface</summary>
@@ -45,7 +45,7 @@
     <id>de.urwpp.Z003</id>
   </suggests>
 
-  <!-- NOTE: Medium urgency of updates is implicitly assumed with AppStream --!>
+  <!-- NOTE: Medium urgency of updates is implicitly assumed with AppStream -->
   <releases>
     <release version="20170801" date="2017-08-01" />
     <release version="20160926" date="2016-09-26" />
diff --git a/appstream/de.urwpp.NimbusSansNarrow.metainfo.xml b/appstream/de.urwpp.NimbusSansNarrow.metainfo.xml
index cdc133b..3db3163 100644
--- a/appstream/de.urwpp.NimbusSansNarrow.metainfo.xml
+++ b/appstream/de.urwpp.NimbusSansNarrow.metainfo.xml
@@ -9,7 +9,7 @@
   <url type="homepage">https://www.urwpp.de/en/</url>
   <url type="bugtracker">https://bugs.ghostscript.com/</url>
 
-  <update_conctact>dkaspar@redhat.com</update_contact>
+  <update_contact>dkaspar@redhat.com</update_contact>
 
   <name>Nimbus Sans Narrow</name>
   <summary>An alternative font family for Helvetica Condensed typeface</summary>
@@ -45,7 +45,7 @@
     <id>de.urwpp.Z003</id>
   </suggests>
 
-  <!-- NOTE: Medium urgency of updates is implicitly assumed with AppStream --!>
+  <!-- NOTE: Medium urgency of updates is implicitly assumed with AppStream -->
   <releases>
     <release version="20170801" date="2017-08-01" />
     <release version="20160926" date="2016-09-26" />
diff --git a/appstream/de.urwpp.P052.metainfo.xml b/appstream/de.urwpp.P052.metainfo.xml
index 32dbb35..c7dc06a 100644
--- a/appstream/de.urwpp.P052.metainfo.xml
+++ b/appstream/de.urwpp.P052.metainfo.xml
@@ -9,7 +9,7 @@
   <url type="homepage">https://www.urwpp.de/en/</url>
   <url type="bugtracker">https://bugs.ghostscript.com/</url>
 
-  <update_conctact>dkaspar@redhat.com</update_contact>
+  <update_contact>dkaspar@redhat.com</update_contact>
 
   <name>P052</name>
   <summary>An alternative font family for Palatino typeface</summary>
@@ -45,7 +45,7 @@
     <id>de.urwpp.Z003</id>
   </suggests>
 
-  <!-- NOTE: Medium urgency of updates is implicitly assumed with AppStream --!>
+  <!-- NOTE: Medium urgency of updates is implicitly assumed with AppStream -->
   <releases>
     <release version="20170801" date="2017-08-01" />
     <release version="20160926" date="2016-09-26" />
diff --git a/appstream/de.urwpp.StandardSymbolsPS.metainfo.xml b/appstream/de.urwpp.StandardSymbolsPS.metainfo.xml
index aaab157..88c102e 100644
--- a/appstream/de.urwpp.StandardSymbolsPS.metainfo.xml
+++ b/appstream/de.urwpp.StandardSymbolsPS.metainfo.xml
@@ -9,7 +9,7 @@
   <url type="homepage">https://www.urwpp.de/en/</url>
   <url type="bugtracker">https://bugs.ghostscript.com/</url>
 
-  <update_conctact>dkaspar@redhat.com</update_contact>
+  <update_contact>dkaspar@redhat.com</update_contact>
 
   <name>Standard Symbols PS</name>
   <summary>An alternative font for Symbol typeface</summary>
@@ -42,7 +42,7 @@
     <id>de.urwpp.Z003</id>
   </suggests>
 
-  <!-- NOTE: Medium urgency of updates is implicitly assumed with AppStream --!>
+  <!-- NOTE: Medium urgency of updates is implicitly assumed with AppStream -->
   <releases>
     <release version="20170801" date="2017-08-01" />
     <release version="20160926" date="2016-09-26" />
diff --git a/appstream/de.urwpp.URWBookman.metainfo.xml b/appstream/de.urwpp.URWBookman.metainfo.xml
index bd3211f..369dc52 100644
--- a/appstream/de.urwpp.URWBookman.metainfo.xml
+++ b/appstream/de.urwpp.URWBookman.metainfo.xml
@@ -9,7 +9,7 @@
   <url type="homepage">https://www.urwpp.de/en/</url>
   <url type="bugtracker">https://bugs.ghostscript.com/</url>
 
-  <update_conctact>dkaspar@redhat.com</update_contact>
+  <update_contact>dkaspar@redhat.com</update_contact>
 
   <name>URW Bookman</name>
   <summary>An alternative font family for ITC Bookman typeface</summary>
@@ -45,7 +45,7 @@
     <id>de.urwpp.Z003</id>
   </suggests>
 
-  <!-- NOTE: Medium urgency of updates is implicitly assumed with AppStream --!>
+  <!-- NOTE: Medium urgency of updates is implicitly assumed with AppStream -->
   <releases>
     <release version="20170801" date="2017-08-01" />
     <release version="20160926" date="2016-09-26" />
diff --git a/appstream/de.urwpp.URWCoreFontSetLevel2.metainfo.xml b/appstream/de.urwpp.URWCoreFontSetLevel2.metainfo.xml
index 0a0c19a..ba2988a 100644
--- a/appstream/de.urwpp.URWCoreFontSetLevel2.metainfo.xml
+++ b/appstream/de.urwpp.URWCoreFontSetLevel2.metainfo.xml
@@ -9,9 +9,9 @@
   <url type="homepage">https://www.urwpp.de/en/</url>
   <url type="bugtracker">https://bugs.ghostscript.com/</url>
 
-  <update_conctact>dkaspar@redhat.com</update_contact>
+  <update_contact>dkaspar@redhat.com</update_contact>
 
-  <!-- NOTE: This is a meta-package to install all the Level 2 fonts --!>
+  <!-- NOTE: This is a meta-package to install all the Level 2 fonts -->
   <name>(URW)++ Core Font Set [Level 2]</name>
   <summary>An alternative font family for New Century Schoolbook typeface</summary>
   <description>
@@ -64,15 +64,15 @@
     <font>Z003 Medium Italic</font>
   </provides>
 
-  <!-- NOTE: No <suggests/> tag here, this metapackages provides all fonts ^--!>
+  <!-- NOTE: No <suggests/> tag here, this metapackages provides all fonts ^-->
 
-  <!-- NOTE: Medium urgency of updates is implicitly assumed with AppStream --!>
+  <!-- NOTE: Medium urgency of updates is implicitly assumed with AppStream -->
   <releases>
     <release version="20170801" date="2017-08-01" />
     <release version="20160926" date="2016-09-26" />
   </releases>
 
-  <!-- NOTE: These font locales are superset of all languages supported --!>
+  <!-- NOTE: These font locales are superset of all languages supported -->
   <languages>
     <lang>aa</lang>
     <lang>af</lang>
diff --git a/appstream/de.urwpp.URWGothic.metainfo.xml b/appstream/de.urwpp.URWGothic.metainfo.xml
index 699f169..9fe308d 100644
--- a/appstream/de.urwpp.URWGothic.metainfo.xml
+++ b/appstream/de.urwpp.URWGothic.metainfo.xml
@@ -9,7 +9,7 @@
   <url type="homepage">https://www.urwpp.de/en/</url>
   <url type="bugtracker">https://bugs.ghostscript.com/</url>
 
-  <update_conctact>dkaspar@redhat.com</update_contact>
+  <update_contact>dkaspar@redhat.com</update_contact>
 
   <name>URW Gothic</name>
   <summary>An alternative font family for ITC Avant Garde Gothic typeface</summary>
@@ -45,7 +45,7 @@
     <id>de.urwpp.Z003</id>
   </suggests>
 
-  <!-- NOTE: Medium urgency of updates is implicitly assumed with AppStream --!>
+  <!-- NOTE: Medium urgency of updates is implicitly assumed with AppStream -->
   <releases>
     <release version="20170801" date="2017-08-01" />
     <release version="20160926" date="2016-09-26" />
diff --git a/appstream/de.urwpp.Z003.metainfo.xml b/appstream/de.urwpp.Z003.metainfo.xml
index b36065f..f1a4cd7 100644
--- a/appstream/de.urwpp.Z003.metainfo.xml
+++ b/appstream/de.urwpp.Z003.metainfo.xml
@@ -9,7 +9,7 @@
   <url type="homepage">https://www.urwpp.de/en/</url>
   <url type="bugtracker">https://bugs.ghostscript.com/</url>
 
-  <update_conctact>dkaspar@redhat.com</update_contact>
+  <update_contact>dkaspar@redhat.com</update_contact>
 
   <name>Z003</name>
   <summary>An alternative font for ITC Zapf Chancery typeface</summary>
@@ -42,7 +42,7 @@
     <id>de.urwpp.URWGothic</id>
   </suggests>
 
-  <!-- NOTE: Medium urgency of updates is implicitly assumed with AppStream --!>
+  <!-- NOTE: Medium urgency of updates is implicitly assumed with AppStream -->
   <releases>
     <release version="20170801" date="2017-08-01" />
     <release version="20160926" date="2016-09-26" />
-- 
2.9.5


From d8acb4f4476e081ff8159d33843f57de7d3a8aee Mon Sep 17 00:00:00 2001
From: "David Kaspar [Dee'Kej]" <dkaspar@redhat.com>
Date: Tue, 20 Feb 2018 13:24:57 +0100
Subject: [PATCH 2/3] appstream: added missing </p> XML tag

  This was causing errors in AppStream parsers, causing the AppStream
  files not being used in software centers/updaters...
---
 appstream/de.urwpp.C059.metainfo.xml              | 1 +
 appstream/de.urwpp.D050000L.metainfo.xml          | 1 +
 appstream/de.urwpp.NimbusMonoPS.metainfo.xml      | 1 +
 appstream/de.urwpp.NimbusRoman.metainfo.xml       | 1 +
 appstream/de.urwpp.NimbusSans.metainfo.xml        | 1 +
 appstream/de.urwpp.P052.metainfo.xml              | 1 +
 appstream/de.urwpp.StandardSymbolsPS.metainfo.xml | 1 +
 appstream/de.urwpp.URWBookman.metainfo.xml        | 1 +
 appstream/de.urwpp.URWGothic.metainfo.xml         | 1 +
 appstream/de.urwpp.Z003.metainfo.xml              | 1 +
 10 files changed, 10 insertions(+)

diff --git a/appstream/de.urwpp.C059.metainfo.xml b/appstream/de.urwpp.C059.metainfo.xml
index 318e115..4c52047 100644
--- a/appstream/de.urwpp.C059.metainfo.xml
+++ b/appstream/de.urwpp.C059.metainfo.xml
@@ -22,6 +22,7 @@
     <p>
       These 35 base fonts are provided freely by (URW++) company,
       and are mainly utilized by Ghostscript, or other applications using it.
+    </p>
   </description>
 
   <provides>
diff --git a/appstream/de.urwpp.D050000L.metainfo.xml b/appstream/de.urwpp.D050000L.metainfo.xml
index dce5db7..292cf23 100644
--- a/appstream/de.urwpp.D050000L.metainfo.xml
+++ b/appstream/de.urwpp.D050000L.metainfo.xml
@@ -22,6 +22,7 @@
     <p>
       These 35 base fonts are provided freely by (URW++) company,
       and are mainly utilized by Ghostscript, or other applications using it.
+    </p>
   </description>
 
   <provides>
diff --git a/appstream/de.urwpp.NimbusMonoPS.metainfo.xml b/appstream/de.urwpp.NimbusMonoPS.metainfo.xml
index 4cdf5e1..a293e23 100644
--- a/appstream/de.urwpp.NimbusMonoPS.metainfo.xml
+++ b/appstream/de.urwpp.NimbusMonoPS.metainfo.xml
@@ -22,6 +22,7 @@
     <p>
       These 35 base fonts are provided freely by (URW++) company,
       and are mainly utilized by Ghostscript, or other applications using it.
+    </p>
   </description>
 
   <provides>
diff --git a/appstream/de.urwpp.NimbusRoman.metainfo.xml b/appstream/de.urwpp.NimbusRoman.metainfo.xml
index a085577..0580780 100644
--- a/appstream/de.urwpp.NimbusRoman.metainfo.xml
+++ b/appstream/de.urwpp.NimbusRoman.metainfo.xml
@@ -22,6 +22,7 @@
     <p>
       These 35 base fonts are provided freely by (URW++) company,
       and are mainly utilized by Ghostscript, or other applications using it.
+    </p>
   </description>
 
   <provides>
diff --git a/appstream/de.urwpp.NimbusSans.metainfo.xml b/appstream/de.urwpp.NimbusSans.metainfo.xml
index 28d9c96..f5ca309 100644
--- a/appstream/de.urwpp.NimbusSans.metainfo.xml
+++ b/appstream/de.urwpp.NimbusSans.metainfo.xml
@@ -22,6 +22,7 @@
     <p>
       These 35 base fonts are provided freely by (URW++) company,
       and are mainly utilized by Ghostscript, or other applications using it.
+    </p>
   </description>
 
   <provides>
diff --git a/appstream/de.urwpp.P052.metainfo.xml b/appstream/de.urwpp.P052.metainfo.xml
index c7dc06a..2504b24 100644
--- a/appstream/de.urwpp.P052.metainfo.xml
+++ b/appstream/de.urwpp.P052.metainfo.xml
@@ -22,6 +22,7 @@
     <p>
       These 35 base fonts are provided freely by (URW++) company,
       and are mainly utilized by Ghostscript, or other applications using it.
+    </p>
   </description>
 
   <provides>
diff --git a/appstream/de.urwpp.StandardSymbolsPS.metainfo.xml b/appstream/de.urwpp.StandardSymbolsPS.metainfo.xml
index 88c102e..134e756 100644
--- a/appstream/de.urwpp.StandardSymbolsPS.metainfo.xml
+++ b/appstream/de.urwpp.StandardSymbolsPS.metainfo.xml
@@ -22,6 +22,7 @@
     <p>
       These 35 base fonts are provided freely by (URW++) company,
       and are mainly utilized by Ghostscript, or other applications using it.
+    </p>
   </description>
 
   <provides>
diff --git a/appstream/de.urwpp.URWBookman.metainfo.xml b/appstream/de.urwpp.URWBookman.metainfo.xml
index 369dc52..c932a29 100644
--- a/appstream/de.urwpp.URWBookman.metainfo.xml
+++ b/appstream/de.urwpp.URWBookman.metainfo.xml
@@ -22,6 +22,7 @@
     <p>
       These 35 base fonts are provided freely by (URW++) company,
       and are mainly utilized by Ghostscript, or other applications using it.
+    </p>
   </description>
 
   <provides>
diff --git a/appstream/de.urwpp.URWGothic.metainfo.xml b/appstream/de.urwpp.URWGothic.metainfo.xml
index 9fe308d..2fc1e7d 100644
--- a/appstream/de.urwpp.URWGothic.metainfo.xml
+++ b/appstream/de.urwpp.URWGothic.metainfo.xml
@@ -22,6 +22,7 @@
     <p>
       These 35 base fonts are provided freely by (URW++) company,
       and are mainly utilized by Ghostscript, or other applications using it.
+    </p>
   </description>
 
   <provides>
diff --git a/appstream/de.urwpp.Z003.metainfo.xml b/appstream/de.urwpp.Z003.metainfo.xml
index f1a4cd7..3d30fd2 100644
--- a/appstream/de.urwpp.Z003.metainfo.xml
+++ b/appstream/de.urwpp.Z003.metainfo.xml
@@ -22,6 +22,7 @@
     <p>
       These 35 base fonts are provided freely by (URW++) company,
       and are mainly utilized by Ghostscript, or other applications using it.
+    </p>
   </description>
 
   <provides>
-- 
2.9.5


From 1f3399b3a798a99ab7fda0b79c09c6ba8cc1822d Mon Sep 17 00:00:00 2001
From: "David Kaspar [Dee'Kej]" <dkaspar@redhat.com>
Date: Wed, 28 Feb 2018 10:27:23 +0100
Subject: [PATCH 3/3] appstream: fix the copy-paste error in summary

  ... in de.urwpp.URWCoreFontSetLevel2.metainfo.xml

  Resolves: #21
---
 appstream/de.urwpp.URWCoreFontSetLevel2.metainfo.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/appstream/de.urwpp.URWCoreFontSetLevel2.metainfo.xml b/appstream/de.urwpp.URWCoreFontSetLevel2.metainfo.xml
index ba2988a..f3bf95b 100644
--- a/appstream/de.urwpp.URWCoreFontSetLevel2.metainfo.xml
+++ b/appstream/de.urwpp.URWCoreFontSetLevel2.metainfo.xml
@@ -13,7 +13,7 @@
 
   <!-- NOTE: This is a meta-package to install all the Level 2 fonts -->
   <name>(URW)++ Core Font Set [Level 2]</name>
-  <summary>An alternative font family for New Century Schoolbook typeface</summary>
+  <summary>Core Font Set containing 35 freely distributable fonts from (URW)++</summary>
   <description>
     <p>
       The Level 2 Core Font Set is a PostScript specification of 35 base fonts
-- 
2.9.5

